### DISCUSSION TITLE
homeassistant install guide

### DISCUSSION THREAD

**Discussion by AndersV209** [tillsteinbach/WeConnect-mqtt#288] [Q&A]
Hello 

is there a quick guide on how to install this on a homeassistant OS setup ? what addons are needed etc.

**Comment by harrymrh** [comment]
Packaging this as a home assistant addon would be the way to go.

**Reply by tillsteinbach** [reply]
Did you have a look at this: https://github.com/mitch-dc/volkswagen_we_connect_id

this is using my python library for home assistant I believe

**Comment by Timer2000** [comment]
This Integration don't support non elektric cars.
